app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -785,6 +789,16 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         }
     }
 
+    override fun getSystemService(name: String): Any? {
+        if(LAYOUT_INFLATER_SERVICE == name){
+            if(inflater == null) {
+                inflater = PerformanceInflater(LayoutInflater.from(baseContext).cloneInContext(this), this)
```

Comment:
Why do we need to clone to pass this in? Add a comment

Reply:
We don't , that was my mistake. Fixed it! Thanks for noticing the redundancy in my code.